CHANGELOG.md

```diff
@@ -114,6 +114,7 @@
 * [ENHANCEMENT] Ingester: do not reuse labels, samples and histograms slices in the write request if there are more entries than 10x the pre-allocated size. This should help to reduce the in-use memory in case of few requests with a very large number of labels, samples or histograms. #10040
 * [ENHANCEMENT] Query-Frontend: prune `<subquery> and on() (vector(x)==y)` style queries and stop pruning `<subquery> < -Inf`. Triggered by https://github.com/prometheus/prometheus/pull/15245. #10026
 * [ENHANCEMENT] Query-Frontend: perform request format validation before processing the request. #10093
+* [ENHANCEMENT] Distributor: Add native histogram support for `electedReplicaPropagationTime` metric in ha_tracker #10264
```

Comment:
this is in the release candidate changelog, can you move it to the unreleased section?

Reply:
Addressed at 
[267a696](https://github.com/grafana/mimir/pull/10264/commits/267a696ad62f4eac1f4372e621f16f4cf8747b1c)